interfaces/cython/cantera/test/test_convert.py

```diff
@@ -717,3 +718,450 @@ def test_ch4_ion(self):
+        ctmlGas, yamlGas = self.checkConversion("explicit-forward-order")
+        self.checkThermo(ctmlGas, yamlGas, [300, 500, 1300, 2000])
+        self.checkKinetics(ctmlGas, yamlGas, [900, 1800], [2e5, 20e5])
+
+    def test_explicit_reverse_rate(self):
+        ctml2yaml.convert(Path(self.test_data_dir).joinpath("explicit-reverse-rate.xml"),
+                          Path(self.test_work_dir).joinpath("explicit-reverse-rate.yaml"))
+        ctmlGas, yamlGas = self.checkConversion("explicit-reverse-rate")
+        self.checkThermo(ctmlGas, yamlGas, [300, 500, 1300, 2000])
+        self.checkKinetics(ctmlGas, yamlGas, [900, 1800], [2e5, 20e5])
+
+    def test_explicit_third_bodies(self):
+        ctml2yaml.convert(Path(self.test_data_dir).joinpath("explicit-third-bodies.xml"),
+                          Path(self.test_work_dir).joinpath("explicit-third-bodies.yaml"))
+        ctmlGas, yamlGas = self.checkConversion("explicit-third-bodies")
+        self.checkThermo(ctmlGas, yamlGas, [300, 500, 1300, 2000])
+        self.checkKinetics(ctmlGas, yamlGas, [900, 1800], [2e5, 20e5])
+
+    def test_fractional_stoich_coeffs(self):
+        ctml2yaml.convert(Path(self.test_data_dir).joinpath("frac.xml"),
+                          Path(self.test_work_dir).joinpath("frac.yaml"))
+        ctmlGas, yamlGas = self.checkConversion("frac")
+        self.checkThermo(ctmlGas, yamlGas, [300, 500, 1300, 2000])
+        self.checkKinetics(ctmlGas, yamlGas, [900, 1800], [2e5, 20e5])
+
+    def test_fixed_chemical_potential_thermo(self):
+        ctml2yaml.convert(Path(self.test_data_dir).joinpath("LiFixed.xml"),
+                          Path(self.test_work_dir).joinpath("LiFixed.yaml"))
+        ctmlGas, yamlGas = self.checkConversion("LiFixed")
+        self.checkThermo(ctmlGas, yamlGas, [300, 500, 1300, 2000])
+
+    def test_water_IAPWS95_thermo(self):
+        ctml2yaml.convert(Path(self.test_data_dir).joinpath("liquid-water.xml"),
+                          Path(self.test_work_dir).joinpath("liquid-water.yaml"))
+        ctmlWater, yamlWater = self.checkConversion("liquid-water")
+        self.checkThermo(ctmlWater, yamlWater, [300, 500, 1300, 2000])
+        self.assertEqual(ctmlWater.transport_model, yamlWater.transport_model)
+        dens = ctmlWater.density
+        for T in [298, 1001, 2400]:
+            ctmlWater.TD = T, dens
+            yamlWater.TD = T, dens
+            self.assertNear(ctmlWater.viscosity, yamlWater.viscosity)
+            self.assertNear(ctmlWater.thermal_conductivity,
+                            yamlWater.thermal_conductivity)
+
+    def test_hmw_nacl_phase(self):
+        ctml2yaml.convert(Path(self.test_data_dir).joinpath("HMW_NaCl_sp1977_alt.xml"),
+                          Path(self.test_work_dir).joinpath("HMW_NaCl_sp1977_alt.yaml"))
+        # ctmlPhase, yamlPhase = self.checkConversion("HMW_NaCl_sp1977_alt")
+        # self.checkThermo(ctmlPhase, yamlPhase, [300, 500])
```

Comment:
Work in progress?

Reply:
Forgot to go change it to `ct.ThermoPhase` as the loader.